packages/apify/src/actor.ts

```diff
@@ -2325,6 +2333,22 @@ export class Actor<Data extends Dictionary = Dictionary> {
         );
     }
 
+    /**
+     * Get time remaining from the Actor run timeout. Returns `undefined` if not on an Apify platform or the current
+     * run was started without a timeout.
+     */
+    private _getRemainingTime(): number | undefined {
+        const env = this.getEnv();
+        if (env.isAtHome === '1' && env.timeoutAt !== null) {
+            return env.timeoutAt.getTime() - Date.now();
+        }
+        log.warning(
+            'Returning `undefined` instead of remaining time. Using `inherit` argument is only possible when ' +
```

Comment:
The user reading this message won't understand what "Returning `undefined`" means. I think we could just remove the first sentence and keep the second one.